Tests/WorkspaceTests/AuthorizationProviderTests.swift

```diff
@@ -34,17 +36,21 @@ final class AuthorizationProviderTests: XCTestCase {
             let authorizationProvider = try configuration.makeAuthorizationProvider(fileSystem: fileSystem, observabilityScope: observability.topScope) as? CompositeAuthorizationProvider
             let netrcProviders = authorizationProvider?.providers.compactMap { $0 as? NetrcAuthorizationProvider }
 
-            XCTAssertEqual(netrcProviders?.count, 1)
-            XCTAssertEqual(try netrcProviders?.first.map { try resolveSymlinks($0.path) }, try resolveSymlinks(customPath))
+            let expectedNetrcProvider = try resolveSymlinks(customPath)
+            #expect(netrcProviders?.count == 1)
+            #expect(try netrcProviders?.first.map { try resolveSymlinks($0.path) } == expectedNetrcProvider)
 
             let auth = authorizationProvider?.authentication(for: "https://mymachine.labkey.org")
```

Comment:
Consider rewriting lines like this as:

```swift
let auth = try #require(authorizationProvider?.authentication(for: "https://mymachine.labkey.org"))
```

Then `auth` won't be optional!

Reply:
ACK.